src/parser/parser_support.c

```diff
@@ -9503,7 +9504,7 @@ pt_make_query_show_collation (PARSER_CONTEXT * parser, int like_where_syntax, PT
  *  specified as LIMIT :offset, :row_count this function returns :offset + :row_count)
  */
 static int
-pt_get_query_limit_from_limit (PARSER_CONTEXT * parser, PT_NODE * limit, DB_VALUE * limit_val)
+pt_get_query_limit_from_limit (PARSER_CONTEXT * parser, PT_NODE * limit, DB_VALUE * limit_val, bool add_offset)
```

Comment:
How about renaming to **pt_get_query_limit_from_limit_clause** for readability?

Reply:
This function is legacy. I'll keep this function name.